web/beacon-app/src/features/onboarding/shared/utils.ts

```diff
@@ -29,3 +29,29 @@ export const getProfessionOptions = () => [
     label: t`Personal`,
   },
 ];
+
+export const getCurrentStepFromMember = (member: any) => {
+  let current = 0;
+  const hasOrganization = member?.organization?.length > 0;
+  const hasWorkspace = member?.workspace?.length > 0;
+  const hasName = member?.name?.length > 0;
+  const hasProfessionSegment = member?.profession_segment?.length > 0;
+
+  if (hasOrganization) {
+    current = 2;
+  }
+  if (hasWorkspace) {
+    current = 3;
+  }
+  if (hasName) {
+    current = 4;
+  }
+  if (hasProfessionSegment) {
+    current = 5; // this step 5 doesnt exist so that will trigger the onboarding complete modal
```

Comment:
Will a modal be displayed once the onboarding process is complete or will the user be directed to the dashboard upon completion? I'm wondering if not displaying a modal could require a change here.

Reply:
Step 5 represents a state of completion. Here, we can show a toast message, open a modal, or even direct the user to the dashboard. Another solution would be to let step 4 redirect the user to the home screen. We will determine the best way to proceed when working on step 4.